Add Raspberry Pi 4 support

- Board: Raspberry PI 4

Ten first thanks for all the help you offer me.

I changed it to pin mode and now it indicates this:
```
2021-01-09 17: 49: 46.112 [ERROR] [core.gpio.pin.digital] Unable to set GPIO26 with cdev, pin is in use
2021-01-09 17: 49: 46.113 [ERROR] [core.gpio.pin.digital] Cannot set GPIO26 with sysfs, pin is in use
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,7 @@
 
 * **Added**
   * Kernel version check to determine GPIO cdev availability and capabilities.
+  * Raspberry Pi 4B support.
 
 * **Fixed**
   * *Unknown* and *Generic Linux* boards missing superclass init.

--- a/mqttany/gpio/boards/rpi.py
+++ b/mqttany/gpio/boards/rpi.py
@@ -34,6 +34,7 @@ from adafruit_platformdetect.constants.boards import (
     RASPBERRY_PI_3A_PLUS,
     RASPBERRY_PI_3B,
     RASPBERRY_PI_3B_PLUS,
+    RASPBERRY_PI_4B,
     RASPBERRY_PI_A,
     RASPBERRY_PI_A_PLUS,
     RASPBERRY_PI_B_PLUS,
@@ -226,6 +227,12 @@ class RPI3Ap(RPI40p):
         self._id = RASPBERRY_PI_3A_PLUS
 
 
+class RPI4B(RPI40p):
+    def __init__(self) -> None:
+        super().__init__()
+        self._id = RASPBERRY_PI_4B
+
+
 SUPPORTED_BOARDS: t.Dict[str, t.Type[Board]] = {
     RASPBERRY_PI_A: RPIA,
     RASPBERRY_PI_B_REV1: RPIBr1,
@@ -238,4 +245,5 @@ SUPPORTED_BOARDS: t.Dict[str, t.Type[Board]] = {
     RASPBERRY_PI_3B: RPI3B,
     RASPBERRY_PI_3B_PLUS: RPI3Bp,
     RASPBERRY_PI_3A_PLUS: RPI3Ap,
+    RASPBERRY_PI_4B: RPI4B,
 }

